biome.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.2.2/schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "recommended": true,
+      "complexity": {
+        "noForEach": "off"
+      }
+    }
+  },
+  "formatter": {
+    "indentStyle": "space",
```

Comment:
Remove this? Tabs are default in biome, didn't want to convert in case someone is strictly opposed to tabs.